rq/worker.py

```diff
@@ -429,6 +429,7 @@ def work(self, burst=False):
 
                 job, queue = result
                 self.execute_job(job)
+                self.set_state('idle')
```

Comment:
This line isn't needed either, because the idle state would have been set in https://github.com/nvie/rq/pull/594/files#diff-1aac421e8c53e895ee40f3b7dc63144fR505 :)